woocommerce/woocommerce
# Add WooCommerce Services Banner to Extensions Page

# Changes in this PR

This PR implements @olaolusoga's designs from p7bbVw-1tf-p2 (under the 'Where else?' section). cc @jeffstieler @nabsul

Here are the current results for reference:

On the featured page:

<img width="1070" alt="wcs-on-featured-extensions-page" src="https://cloud.githubusercontent.com/assets/11487924/26567971/9f302c4a-44cd-11e7-8064-a67d501e63cc.png">

On the shipping page:

<img width="1200" alt="wcs-on-shipping-extensions-page" src="https://cloud.githubusercontent.com/assets/11487924/26567979/b2f475c4-44cd-11e7-9383-7021dea68404.png">

# Testing Instructions

1. In `/includes/admin/class-wc-admin-addons.php` make sure to use the test json files (I hosted the files locally):

- on line 27 use this [featured json file](http://cld.wthms.co/YH7mXT)
- on line 49, use this [sections json file](http://cld.wthms.co/lp0nia)
- use this different [shipping json file](http://cld.wthms.co/ZtnNhx), in the sections json file above, make sure to replace line 16 with the location of your shipping file

2. Make sure to delete your WordPress transients before testing, you can use a plugin like [Transient Cleaner](https://wordpress.org/plugins/artiss-transient-cleaner/)
3. Go to `wp-admin/admin.php?page=wc-addons&section=featured` and `wp-admin/admin.php?page=wc-addons&section=shipping_methods` to see the changes

# Questions
- what do you think of the install URL for the button? It goes to `http://wp.dev/wp-admin/plugin-install.php?tab=plugin-information&plugin=woocommerce-services`

# Todos
- [ ] we need to add the responsive designs
- [ ] we need a hover state for the button
- [ ] blank target for install URL
- [ ] fix banner alignment on the non-featured pages. it needs to align with the product sections below at different screen sizes
